WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -782,6 +869,25 @@ - (void)syncContentEnded
 }
 
 
+#pragma mark - Async Loading Helpers
+
+- (void)setupWithPostID:(NSNumber *)postID siteID:(NSNumber *)siteID
+{
+    NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
+    ReaderPostService *service      = [[ReaderPostService alloc] initWithManagedObjectContext:context];
+    __weak __typeof(self) weakSelf  = self;
+    
+    [service fetchPost:postID.integerValue forSite:siteID.integerValue success:^(ReaderPost *post) {
```

Comment:
I'll nitpick here but this is a comment that applies to a few other places.  I think dot syntax is great when we're dealing with properties with getters and setters but I have mixed feelings about dot notations with methods.  You can do it, and it works, but i think it adds some semantic confusion: is the thing being accessed a method or a property. 
Thoughts?